source/ui/Select.js

```diff
@@ -62,7 +62,34 @@
 			* @default null
 			* @public
 			*/
-			value: null
+			value: null,
+
+			/**
+			* The size of the select box in rows.
+			* 
+			* @type {Number}
+			* @default 1
+			* @public
+			*/
+			size: 1,
```

Comment:
Should this be `0` to match the default specified in the spec: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select#Attributes.